### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by TheiLLeniumStudios**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -11,41 +11,36 @@ AddEventHandler('AttackTransport:akceptujto', function()
...
 
-		OdpalTimer()
-    else
-		TriggerClientEvent('QBCore:Notify', _source, 'Need at least '..ActivePolice.. ' SASP to activate the mission.')
-    end
+			OdpalTimer()
+		else
+			TriggerClientEvent('QBCore:Notify', _source, 'Need at least '..ActivePolice.. ' SASP to activate the mission.')
+		end
+		end
```
Comment (BerkieBb) [inline server/main.lua:35]: This doesn't seem like it is indentated right

File: server/main.lua [hunk 2]
```diff
@@ -61,7 +56,7 @@ AddEventHandler('AttackTransport:zawiadompsy', function(x ,y, z)
 end)
 
 RegisterServerEvent('AttackTransport:graczZrobilnapad')
-AddEventHandler('AttackTransport:graczZrobilnapad', function(moneyCalc)
+AddEventHandler('AttackTransport:graczZrobilnapad', function(_)
```
Comment (BerkieBb) [inline server/main.lua]: You can remove this underscore completely

Comment (BerkieBb) [reply server/main.lua]:
```suggestion
AddEventHandler('AttackTransport:graczZrobilnapad', function()
```

File: server/main.lua (hunk 1 above)
Comment (MatthewRorke) [reply server/main.lua:35]: Resolved

**Review by MatthewRorke — APPROVED** [review]
Indentation concerns resolved. Script functions as intended.

**Review by BerkieBb — APPROVED** [review]

**Pull request merged**